configuration/environments.rst

```diff
@@ -20,206 +20,136 @@ Different Environments, different Configuration Files
-        }
-    }
-
-As you can see, when Symfony is loaded, it uses the given environment to
-determine which configuration file to load. This accomplishes the goal of
-multiple environments in an elegant, powerful and transparent way.
-
-Of course, in reality, each environment differs only somewhat from others.
-Generally, all environments will share a large base of common configuration.
-Opening the ``config_dev.yml`` configuration file, you can see how this is
-accomplished easily and transparently:
-
-.. configuration-block::
-
-    .. code-block:: yaml
-
-        imports:
-            - { resource: config.yml }
-
-        # ...
-
-    .. code-block:: xml
-
-        <?xml version="1.0" encoding="UTF-8" ?>
-        <container xmlns="http://symfony.com/schema/dic/services"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xmlns:doctrine="http://symfony.com/schema/dic/doctrine"
-            xsi:schemaLocation="http://symfony.com/schema/dic/services
-                http://symfony.com/schema/dic/services/services-1.0.xsd
-                http://symfony.com/schema/dic/doctrine
-                http://symfony.com/schema/dic/doctrine/doctrine-1.0.xsd">
-
-            <imports>
-                <import resource="config.xml" />
-            </imports>
-
-            <!-- ... -->
-
-        </container>
-
-    .. code-block:: php
-
-        $loader->import('config.php');
-
-        // ...
-
-To share common configuration, each environment's configuration file
-simply first imports from a central configuration file (``config.yml``).
-The remainder of the file can then deviate from the default configuration
-by overriding individual parameters. For example, by default, the ``web_profiler``
-toolbar is disabled. However, in the ``dev`` environment, the toolbar is
-activated by modifying the value of the ``toolbar`` option in the ``config_dev.yml``
-configuration file:
-
-.. configuration-block::
-
-    .. code-block:: yaml
-
-        # app/config/config_dev.yml
-        imports:
-            - { resource: config.yml }
-
-        web_profiler:
-            toolbar: true
-            # ...
-
-    .. code-block:: xml
-
-        <!-- app/config/config_dev.xml -->
-        <?xml version="1.0" encoding="UTF-8" ?>
-        <container xmlns="http://symfony.com/schema/dic/services"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xmlns:webprofiler="http://symfony.com/schema/dic/webprofiler"
-            xsi:schemaLocation="http://symfony.com/schema/dic/services
-                http://symfony.com/schema/dic/services/services-1.0.xsd
-                http://symfony.com/schema/dic/webprofiler
-                http://symfony.com/schema/dic/webprofiler/webprofiler-1.0.xsd">
-
-            <imports>
-                <import resource="config.xml" />
-            </imports>
+            // ...
+            $confDir = dirname(__DIR__).'/config';
```

Comment:
`$this->getProjectDir()`? in line with the latest changes https://github.com/symfony/recipes/pull/244.